W3ChampionsStatisticService/Hubs/WebsiteBackendHub.cs

```diff
@@ -276,6 +276,33 @@ await Clients.Caller.SendAsync(
         }
     }
 
+    public async Task BlockPlayer(string battleTag)
+    {
+        var currentUser = _connections.GetUser(Context.ConnectionId)?.BattleTag;
+        if (currentUser == null)
+        {
+            return;
+        }
+        try
+        {
+            var friendList = await _friendCommandHandler.LoadFriendList(currentUser);
+            CanBlock(friendList, battleTag);
+            friendList.BlockedBattleTags.Add(battleTag);
```

Comment:
Note that this here is teh WebsiteBackendHub, speak an API Handler. You'd generally want function in this file to not have business logic but only call downstream handlers that actually do the logic. So the adding to the blocklist etc should all be the downstream handlers and not this API handlers.

Reply:
I'm just matching the existing patterns, I have no interest in refactoring this for the scope of this change.